tutorial/20_recipes/002_multi_objective.py

```diff
@@ -0,0 +1,71 @@
+"""
+.. _multi_objective_lgbm:
+
+Multi-objective optimization example with LightGBM
+==================================================
+
+This tutorial is an example of multi-objective optimization with Optuna which optimizes
+precision and recall while it would be more common to use F-measure and (ROC-)AUC instead.
+
+As of `v2.4.0 <https://github.com/optuna/optuna/releases/tag/v2.4.0>`_, :class:`~optuna.multi_objective.MultiObjectiveStudy` is merged into :class:`~optuna.study.Study`.
+"""
+
+
+###################################################################################################
+# First of all, import requirements.
+import lightgbm as lgb
+import numpy
+from sklearn import datasets
+from sklearn.metrics import precision_score, recall_score
+from sklearn.model_selection import train_test_split
+
+import optuna
+
+
+###################################################################################################
+# In the ``objective`` below, we train a model using Breast cancer dataset before
+# calculating precision and recall.
+def objective(trial):
+    data, target = datasets.load_breast_cancer(return_X_y=True)
+    train_x, valid_x, train_y, valid_y = train_test_split(data, target, test_size=0.25)
+    dtrain = lgb.Dataset(train_x, label=train_y)
+
+    param = {
+        "objective": "binary",
+        "metric": "binary_logloss",
+        "verbosity": -1,
+        "boosting_type": "gbdt",
+        "lambda_l1": trial.suggest_float("lambda_l1", 1e-8, 10.0, log=True),
+        "lambda_l2": trial.suggest_float("lambda_l2", 1e-8, 10.0, log=True),
+        "num_leaves": trial.suggest_int("num_leaves", 2, 256),
+        "feature_fraction": trial.suggest_float("feature_fraction", 0.4, 1.0),
+        "bagging_fraction": trial.suggest_float("bagging_fraction", 0.4, 1.0),
+        "bagging_freq": trial.suggest_int("bagging_freq", 1, 7),
+        "min_child_samples": trial.suggest_int("min_child_samples", 5, 100),
+    }
+
+    gbm = lgb.train(param, dtrain)
+    preds = gbm.predict(valid_x)
+    pred_labels = numpy.rint(preds)
+    precision = precision_score(valid_y, pred_labels)
+    recall = recall_score(valid_y, pred_labels)
+    return precision, recall
+
+
+###################################################################################################
+# To make :class:`~optuna.study.Study` compatible with multi-objective optimization,
+# what we need to do is only to pass the collection of ``"direction"``\\s.
```

Comment:
Seems like the backslash isn't escaped properly. Could you have a look? https://65701-122299416-gh.circle-artifacts.com/0/docs/build/html/tutorial/20_recipes/002_multi_objective.html#sphx-glr-tutorial-20-recipes-002-multi-objective-py

Reply:
I think we just need to remove the backslash here.
```suggestion
# what we need to do is only to pass the collection of ``"direction"``s.
```